frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,168 @@
+  }, 2 * MINUTE);
+
+  it(
+    'tests subscription flow for ocs operator',
+    async () => {
+      await Installer.subscribeToOperator();
+      const text = await ocsOperatorStatus.getText();
+      expect(text.includes('Succeeded')).toBe(true);
+    },
+    2 * MINUTE,
+  );
+
+  it('tests for presence of 3 operator pods', async () => {
+    const podList = JSON.parse(
+      execSync('kubectl get po -n openshift-storage -o json').toString('utf-8'),
+    );
+    const pods = podList.items;
+    let pod = getPodData(pods, POD_NAME_PATTERNS.OCS);
+    let phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    pod = getPodData(pods, POD_NAME_PATTERNS.ROOK);
+    phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    pod = getPodData(pods, POD_NAME_PATTERNS.NOOBA_OPERATOR);
+    phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+  });
+});
+
+describe('Test creation of Storage Cluster', () => {
+  it(
+    'creates a storage cluster',
+    async () => {
+      const nodes = await Installer.createStorageCluster();
+      const text = await crudView.resourceTitle.getText();
+      expect(text).toEqual(STORAGE_CLUSTER_NAME);
+      // Wait for cluster to come to ready state
+      await checkIfClusterIsReady();
+      // Verify all the nodes have the required labels
+      nodes.forEach((node) => {
+        expect(verifyNodeLabels(node, OCS_NODE_LABEL)).toBe(true);
+      });
+    },
+    15 * MINUTE,
+  );
+});
+
+describe('Tests for pods and storage classes', () => {
+  let pods = null;
+
+  beforeAll(async () => {
+    const podList = await JSON.parse(
+      execSync('kubectl get po -n openshift-storage -o json').toString('utf-8'),
+    );
+    pods = podList.items;
+  });
+
+  it('tests if ocs operator pods is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.OCS);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-operator is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.ROOK);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if nooba-operator is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.NOOBA_OPERATOR);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if nooba-core is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.NOOBAA_CORE);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mon-a is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.ROOK_CEPH_MON_A);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mon-b is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.ROOK_CEPH_MON_B);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mon-c is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.ROOK_CEPH_MON_C);
+    testPodIsRunning(getPodPhase(pod));
+  });
```

Comment:
It's not required that it will always be `mon-[a/b/c]`. It could be `mon-[a/b/d]` or `mon-[a/d/e]` and so on.
I think a check to verify that there are at least 3 Mons created is good enough.

Reply:
ack